packages/remote-config/lib/index.d.ts

```diff
@@ -278,20 +297,80 @@ export namespace FirebaseRemoteConfigTypes {
    */
   export class Module extends FirebaseModule {
     /**
-     * The number of milliseconds since the last Remote RemoteConfig fetch was performed.
+     * Indicates the value in milliseconds set for the minimum interval that needs to elapse
+     * before a fetch request can again be made to the Remote Config backend.
      */
-    lastFetchTime: number;
+    minimumFetchIntervalMillis: number;
     /**
-     * Whether developer mode is enabled. This is set manually via {@link config#setConfigSettings}
+     * Indicates the default value in milliseconds to abandon a pending fetch request made to the Remote Config backend.
      */
-    isDeveloperModeEnabled: boolean;
+    fetchTimeMillis: number;
+    /**
+     * The number of milliseconds since the last Remote RemoteConfig fetch was performed.
+     */
+    lastFetchTime: number;
```

Comment:
This is `fetchTimeMillis` on the web sdk: https://firebase.google.com/docs/reference/js/firebase.remoteconfig.RemoteConfig#fetchtimemillis